services/game-actions.js

```diff
@@ -0,0 +1,45 @@
+import firebase from 'firebase'
+
+export function gameUpdate(place, year) {
+  return new Promise((resolve, reject) => {
+    firebase.database().ref(`games/${game.id}`).set({
+      place: game.place,
+      year: game.year,
+      //time: game.time,
```

Comment:
Make sure to include `timestamp: game.timestamp` for this object. It's really important so we know which games to delete in the database